tests/test_core.py

```diff
@@ -8302,7 +8302,7 @@ def test_pthread_dylink(self):
     side = path_from_root('tests', 'core', 'pthread', 'test_pthread_dylink_side.c')
     self.dylink_testf(main, side, "success", need_reverse=False)
 
-  @needs_dlfcn
```

Comment:
This one puzzles me, I don't know why this particular test function didn't get updated in my fork when I clearly see it updated in the github source.

Reply:
Sorry, that a bad commit on my part.  Should be fixed now.